lib/modules/platform/gerrit/client.ts

```diff
@@ -219,6 +219,10 @@ class GerritClient {
     return message.substring(0, 0x4000).trim();
   }
 
+  private static removeQuotes(message: string): string {
+    return message.replace(/"/g, '');
```

Comment:
inline, it's only used once, also use regEx helper function

Reply:
Will do! Thanks